compiler/modules/types/core.bal

```diff
@@ -119,11 +120,11 @@ public readonly class ComplexSemType {
     // For a uniform type with code c,
     // all & (1 << c) is non-zero iff this type contains all of the uniform type
     // some & (1 << c) is non-zero iff this type contains some but not all of the uniform type
-    UniformTypeBitSet all;
-    UniformTypeBitSet some;
+    public UniformTypeBitSet all;
```

Comment:
Making these things public is not OK. The public interface exposed by ComplexSemType should be type-safe